modules/members.js

```diff
@@ -0,0 +1,101 @@
+// @ts-check
+
+const Augur = require("augurbot-ts"),
+  u = require("../utils/utils"),
+  c = require("../utils/modCommon"),
+  Discord = require("discord.js"),
+  config = require("../config/config.json");
+
+/**
+ * Creates a profile card - a PNG that contains some user information in a fun format!
+ * @param {Discord.GuildMember} member The member to create the profile card for.
+ * @returns {Promise<Buffer|undefined>} File-like object to attach to your response.
+ */
+async function makeProfileCard(member) {
+  try {
+    const badgeData = require("../utils/badges"),
+      RankInfo = require("../utils/RankInfo"),
+      Jimp = require("jimp");
+
+    const badgePath = "./media/badges/";
+    const font = await Jimp.loadFont(Jimp.FONT_SANS_16_WHITE);
```

Comment:
I don't think we need to load the font every time someone runs the command. Save some time by defining it early in the file and setting it in the init. Inits usually finish before the ready event, so the command shouldn't have the chance to run before the font is loaded.

```js
/** @type {Jimp.font} */
let font;
...
Module.setInit(async () => {
  font = await Jimp.loadFont()
})
```

Reply:
Set it to be initialized at init